fixtures/integrations/jira/stubs/createmeta_response.json

```diff
@@ -125,6 +128,7 @@
                 "customId": 10202
```

Comment:
Can you educate me on what the fields in the custom fields are? Just by looking at the previous ones it seems like customId is the same number that comes after the field which is `10300` in this case. But I have no idea how these are used.

Reply:
So I'm not 100% sure about the full schema, but these are the fields we care about:
- `schema.custom` - some namespaced identifier for the source of the custom field. Many of these seem to be plugin/add-on namespaces such as:
    - `com.atlassian.jira.plugin.system.customfieldtypes:datepicker` - date field
    - `com.pyxis.greenhopper.jira:gh-lexo-rank` - ranking field
-  `key` - We use this when creating the issue payload to send to Jira, which acts as the property name more or less.

Now for the confusing one:
`schema.customId` - Appears to be some incrementing ID field on the issue that's generated when you add it to the template? Luckily, I haven't found a need to reference this.

It's very similar to `key`, and in my own testing seems to align with the key's`customfield_<number>` identifier, although I don't think there's any guarantee on this as Jira's own API examples don't align: https://developer.atlassian.com/cloud/jira/platform/rest/v3/api-group-issue-fields/#api-rest-api-3-field-search-get.

Reply:
Thanks for explaining. Yeah wanted to make sure it doesn't have to align.